From b05c35217177bea72e71455644b182817049462f Mon Sep 17 00:00:00 2001
From: Radha Mohan Chintakuntla <rchintakuntla@cavium.com>
Date: Wed, 14 Feb 2018 09:07:05 -0800
Subject: [PATCH 0010/1051] mtd: nand: cavium: Rename nand.h to rawnand.h

The upstream code renamed the header file as part of cleanup.

Signed-off-by: Radha Mohan Chintakuntla <rchintakuntla@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/mtd/nand/raw/cavium_nand.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/nand/raw/cavium_nand.c b/drivers/mtd/nand/raw/cavium_nand.c
index bfa822071724..8e12cf0a4856 100644
--- a/drivers/mtd/nand/raw/cavium_nand.c
+++ b/drivers/mtd/nand/raw/cavium_nand.c
@@ -17,7 +17,7 @@
 #include <linux/mm.h>
 #include <linux/module.h>
 #include <linux/mtd/mtd.h>
-#include <linux/mtd/nand.h>
+#include <linux/mtd/rawnand.h>
 #include <linux/mtd/nand_bch.h>
 #include <linux/mtd/nand_ecc.h>
 #include <linux/of.h>
-- 
2.17.1

